MainSite/views.py

```diff
@@ -27,12 +29,19 @@ def index(request):
     kandz = var[3]
     eppely = var[4]
 
+    totalPower = float(enphasePower) + float(froniusPower) + float(windPower)
+
     with connection.cursor() as cur:
         cur.execute(sql2)
         var = cur.fetchone()
 
     selPower = var[1]
 
+    with connection.cursor() as cur:
+        cur.execute(sql3)
+        var = cur.fetchone()
```

Comment:
Consider jumping to using an ORM sooner rather than later. The more code you write without it, the more painful it may be to switch, but the ORM is definitely a more understandable API to accessing the database.